modules/packages/Sort.chpl

```diff
@@ -422,13 +422,14 @@ proc radixSortOk(Data: [?Dom] ?eltType, comparator) param {
 
 /*
 
-Sort the elements in a 1D rectangular array.  The choice of sorting
-algorithm used is made by the implementation.
+Sort the elements in a 1D rectangular array. After the call, the ``Data`` array
+will store elements n sorted order.
+
+The choice of sorting algorithm used is made by the implementation.
 
 .. note::
 
-  This function does not run a stable sort. Elements that compare
-  the same can be rearranged by this call.
+  Elements will be moved, swapped, or assigned in to place.
```

Comment:
I'll update the wording to say "will be reordered ..." as you suggest.

I don't want to add a `param useAssignment` until we have a specific use case for it. It would be speculative work that requires changes throughout the module.

Currently, the quicksort pivots are move'd around. But you have a good point that I should mention that copy initialization is possible there.